experiments/AtmosLES/stable_bl_model.jl

```diff
@@ -192,8 +197,9 @@ function init_problem!(problem, bl, state, aux, localgeo, t)
     state.ρ = ρ
     state.ρu = SVector(ρu, ρv, ρw)
     state.ρe = ρe_tot
-    state.moisture.ρq_tot = ρ * q_tot
-
+    if !(bl.moisture isa DryModel)
+        state.moisture.ρq_tot = ρ * q_tot
+    end
     if z <= FT(50) # Add random perturbations to bottom 50m of model
         state.ρe += rand() * ρe_tot / 100
     end
```

Comment:
you are missing a call or initialize the EDMF here 
"init_state_prognostic!(bl.turbconv, bl, state, aux, localgeo, t)"